package.json

```diff
@@ -25,9 +25,9 @@
     "prepare": "yarn build",
     "start": "yarn storybook",
     "storybook": "start-storybook -p 3000 -s public/development",
-    "test": "cross-env TZ=America/Chicago NODE_OPTIONS='--max-old-space-size=8096' jest --detectOpenHandles",
-    "test:a11y": "cross-env TZ=America/Chicago jest --testPathPattern='src\\/([^\\s]+)\\/([a-zA-Z0-9\\s_\\\\.\\-\\(\\):])+(\\.test\\.a11y\\.)+(js|jsx)$'",
-    "test:watch": "cross-env TZ=America/Chicago NODE_OPTIONS='--max-old-space-size=8096' jest  --watch --verbose",
+    "test": "cross-env TZ=America/Chicago NODE_OPTIONS='--max-old-space-size=8096' jest --testPathPattern='.*\\.test\\.js(x)?' --detectOpenHandles --coverage",
```

Comment:
`--detectOpenHandles` can be removed here. This will also enable this PR to close #732, the PR body will need to be updated to auto-close it on merge.